src/main/kotlin/org/ionproject/integration/step/tasklet/iseltimetable/implementations/TimetableTasklet.kt

```diff
@@ -0,0 +1,26 @@
+package org.ionproject.integration.step.tasklet.iseltimetable.implementations
+
+import org.ionproject.integration.job.ISELTimetable
+import org.ionproject.integration.utils.JsonUtils
+import org.ionproject.integration.utils.Try
+import org.ionproject.integration.utils.orThrow
+import org.slf4j.LoggerFactory
+import org.springframework.batch.core.StepContribution
+import org.springframework.batch.core.scope.context.ChunkContext
+import org.springframework.batch.core.step.tasklet.Tasklet
+import org.springframework.batch.repeat.RepeatStatus
+import org.springframework.stereotype.Component
+
+@Component
+class TimetableTasklet(private val state: ISELTimetable.State) :
+    Tasklet {
+    private val log = LoggerFactory.getLogger(TimetableTasklet::class.java)
+    override fun execute(contribution: StepContribution, chunkContext: ChunkContext): RepeatStatus? {
+
+        val json = Try.ofValue(state.timetableTeachers.timetable.toTypedArray())
+            .flatMap { t -> JsonUtils.toJson(t) }
+
+        log.info(json.orThrow())
```

Comment:
We cannot do this. Every log lines could be deleted without changing the app behaviour, which doesn't happen in this case.